FrostyCli/README.md

```diff
@@ -64,4 +64,8 @@ Or as an alternative to the datapath launch command, you can use the `GAME_DATA_
 ```GAME_DATA_DIR=<mod data path>```
 
 
-
+> [!NOTE]
+> For Linux users modding games of Frostbite version above 2014.4.11 (check Frosty server pinned messages or Info_Win64_retail.BuildSettings if available) 
```

Comment:
Check the <profile>.json file in Profiles folder, also add that windows user need to get the CryptBase.dll from v1 or is that already somewhere?

Reply:
ah I forgot about cryptbase, I'll add that to thirdparty as well and mention it then